### PR TITLE
Render fetch_maven_artifacts plugin

### PR DESCRIPTION
**Pull request by lcarva**
Signed-off-by: Luiz Carvalho <lucarval@redhat.com>

### CODE REVIEW

**Comment by rcerven** [comment]
missing change in api, to pass the artifacts_allowed_domains to spec

**Comment by lcarva** [comment]
@rcerven added change to api.py, thanks!

File: inputs/prod_inner.json [hunk 1]
```diff
@@ -66,6 +66,13 @@
     },
     {
       "args": {
+        "koji_root": "{{KOJI_ROOT}}",
+        "koji_hub": "{{KOJI_HUB}}"
+      },
+      "name": "fetch_maven_artifacts"
```
Comment (twaugh) [inline inputs/prod_inner.json]: Do we want `"required": false`, to retain compatibility with atomic-reactor-1.6.18?

Comment (lcarva) [inline inputs/prod_inner.json]: Yes, great idea! Added.

File: osbs/conf.py [hunk 2]
```diff
@@ -491,3 +491,11 @@ def get_can_orchestrate(self):
     def get_info_url_format(self):
         return self._get_value("info_url_format", self.conf_section,
                                "info_url_format")
+
+    def get_artifacts_allowed_domains(self):
+        value = self._get_value("artifacts_allowed_domains", self.conf_section,
+                                "artifacts_allowed_domains")
+        if value:
+            return [x.strip() for x in ''.join(value.splitlines()).split(',')]
```
Comment (twaugh) [inline osbs/conf.py]: What's the `splitlines()` part for? Isn't `[x.strip() for x in value.split(',')]` sufficient?

Comment (vrutkovs) [inline osbs/conf.py]: That's required to handle line breaks: ```\n  spam.com,     eggs.com,\n\n bacon.co```

Comment (lcarva) [inline osbs/conf.py]: Yes, that's much cleaner. Changed accordingly. Also, created a new commit for using the same approach for parsing `smtp_additional_addresses` and `smtp_error_addresses`.

**Pull request merged**